0 Products/80 OpenGrab EPM v3/0 Main Page.md

```diff
@@ -2,6 +2,12 @@
 
 ## Overview
 
+V3R5C
+
+<img src="EPMV3R5C.jpg" class="thumbnail" title="Top view">
+
+V3R4B
+
```

Comment:
I take it you didn't try to render that. Please render your changes using the script `run.py`; this source will look horribly wrong. On the main page:

![image](https://cloud.githubusercontent.com/assets/3298404/19872304/233fa898-9fca-11e6-8258-0e5dc4841fa3.png)

The above happened because the site engine automatically picks up the first paragraph as a brief description. 

The page itself:

![image](https://cloud.githubusercontent.com/assets/3298404/19872330/39aa9516-9fca-11e6-88bf-0e02aa980ab6.png)

You want to describe a picture, so using a paragraph for that is fundamentally wrong. Use the HTML attribute `title` instead:

```html
<img src="EPMV3R5C.jpg" class="thumbnail" title="Version V3R5C, top view">
```

Maybe the title itself is not enough if you want to emphasize what each revision should look like. In that case you could probably use sub-headings or something, first having explained that these pictures are of different revisions of the same device. I guarantee you, a [*drunk user*](https://www.youtube.com/watch?v=r2CbbBLVaPk) will have no idea what does `V3R5C` and `V3R4B` mean.

Also note that the picture titled as `image.jpg` will be automatically used as a thumbnail on the main page. Probably it makes sense to show the *newest* revision on the thumbnail rather than the older one.